torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -513,34 +512,33 @@ void LLVMCodeGenImpl::emitKernel(
 
   irb_.CreateRet(value_);
 
-#if DEBUG_PRINT
-  llvm::errs() << *module_;
-#endif
   if (llvm::verifyFunction(*fn_, &llvm::outs())) {
     throw std::runtime_error("Function verification failed");
   }
 
-  // print graph debug info.
-  std::string fnstr;
-  llvm::raw_string_ostream FS(fnstr);
-  fn_->print(FS);
```

Comment:
This print has disappeared, is it duplicating what's printed in the module?

Reply:
Yes, it's a duplicate. `module_->print()` prints the graph info generated by `fn_->print()` plus info associated with the module; it should be more helpful.